frontend/package.json

```diff
@@ -15,11 +15,10 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^5.3.0",
     "recharts": "^2.10.3",
+    "vite-plugin-commonjs": "^0.10.1",
     "web-vitals": "^2.1.4"
   },
-  "engines": {
-    "node": "21.0.0"
-  },
+  
```

Comment:
I understand that you are trying to host static files, which is what we SHOULD be doing, but we thought that should be a specific ticket on its own to isolate changes in a small way and then verify it works as expected.

Reply:
You are right.